connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/ArtifactManager.scala

```diff
@@ -48,7 +48,7 @@ import org.apache.spark.util.{ThreadUtils, Utils}
  *   An unique identifier of the session which the artifact manager belongs to.
  * @param channel
  */
-class ArtifactManager(
+private[sql] class ArtifactManager(
```

Comment:
@vicennial Should the artifact and UDF related classes default to be public or package protected?